core/schema/project/asset_other.yml

```diff
@@ -5,7 +5,7 @@ type: "shotgun_entity"
 entity_type: "Asset"
 
 # the shotgun field to use for the folder name
-name: "{sg_asset_type}/{sg_asset_category}/{code}"
+name: "{sg_asset_type}/{sg_asset_library}/{sg_asset_category}/{code}"
```

Comment:
this order of folders (type/library/category) is different from the asset.yml (library/type/category). Do we want to unify this ordering?

Reply:
> this order of folders (type/library/category) is different from the asset.yml (library/type/category). Do we want to unify this ordering?

That's actually intentional, so that the few special AssetTypes such as Publishing, Tools, Engine, etc can have a top-level folder next to but separate from the Content folder. However, we want some of those assets (such as Publishing type) to be part of an existing AssetLibrary that is used for game Content, without that AssetLibrary folder name appearing at the root. In other words, if "Gen2" is an AssetLibrary, the folders would look like this:

Content/Gen2/AssetCategory/...
Publishing/Gen2/AssetCategory/...